deploy.sh

```diff
@@ -9,10 +9,8 @@ services=(
   'uploads'
   'app-api'
   'stream-functions'
-  'ui-waflog-s3-bucket'
   'ui'
   'ui-auth'
-  'ui-waf-log-assoc'
   'ui-src'
```

Comment:
These services were collapsed into the 'ui' service.  I'll explain the history here:
ui-waflog-s3-bucket:  The idea in making this a separate service was to decouple the bucket creation and the configuration that used it (ui service).  Why decouple?  CMS projects are typically assigned a bucket to write logs to, and do not want to create their own.  By decoupling at the service level, we felt a CMS project could choose not to deploy the ui-waflog-s3-bucket service, and modify a custom variable in the ui service, and be done.  In practice, this hasn't helped anybody, it's difficult to explain, it costs us performance as a separate service, and is (in my view) no easier than removing the AWS::S3::Bucket resource from the ui service if you choose.  The creation of the S3 bucket has been moved to the ui service.
ui-waf-log-assoc:  This associates a firehose stream with a waf webacl.  This can't be done in cloudformation yet.  The reason this was made into a separate service was simple... we didn't have a great way to fetch the firehose and webacl arn's until after they were created.  That's the only reason this was separate.  By modifying the aws cli script that makes the association to lookup those arn's, I was able to move the association functionality to the ui service.  So, the assoc service was removed.